src/analysis/polarity_search.ml

```diff
@@ -122,7 +122,7 @@ let directories ~global_modules env =
 
 let execute_query query env dirs =
   let direct dir acc =
-    Env.fold_values (fun _ path desc acc ->
+    Env.fold_values (fun _ path desc _ acc ->
```

Comment:
I'm not sure if it's appropriate to ignore modalities here - I don't know what this file does.